cmd/nerdctl/stats_linux_test.go

```diff
@@ -32,11 +33,17 @@ func TestStats(t *testing.T) {
 		t.Skip("test skipped for rootless containers on cgroup v1")
 	}
 	testContainerName := testutil.Identifier(t)
+	exitedTestContainerName := fmt.Sprintf("%s-exited", testContainerName)
+	if len(testContainerName) > 12 {
+		testContainerName = testContainerName[:12]
```

Comment:
`t.Parallel()` is a bit risky here, because we are truncating the `testContainerName`, it may accidentally delete other test container